src/utils/efd.c

```diff
@@ -119,6 +119,7 @@ int nn_efd_wait (struct nn_efd *self, int timeout)
     }
 
     for (;;) {
+        fd = self->r;
```

Comment:
Agreed.  I still haven’t figured it out.  This is going to take time.

On Wed, May 11, 2016 at 4:33 PM, Jack R. Dunaway <notifications@github.com>
wrote:

> In src/utils/efd.c
> <https://github.com/nanomsg/nanomsg/pull/703#discussion_r62944653>:
>
> > @@ -119,6 +119,7 @@ int nn_efd_wait (struct nn_efd *self, int timeout)
> >      }
> >
> >      for (;;) {
> > +        fd = self->r;
>
> Good catch; even if doesn't fix issue, this is a good change to keep.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nanomsg/nanomsg/pull/703/files/f19915e9adcbf4079825205de1c56d367ccacc07#r62944653>
>